### PR TITLE
3.4 Change Example to use allControls instead of allInputs

### PR DESCRIPTION
**Pull request by inoas**

### CODE REVIEW

**Comment by inoas** [comment]
Ref: https://github.com/cakephp/cakephp/pull/9944

File: en/views/helpers/form.rst [hunk 1]
```diff
@@ -1767,7 +1767,7 @@ control the generated legend/fieldset.
 
 For example::
 
-    echo $this->Form->allInputs(
+    echo $this->Form->allControls(
```
Comment (antograssiot) [inline en/views/helpers/form.rst]: Keeping both versions with  version note is better IMO and in line with most of what has been done in similar cases

Comment (inoas) [inline en/views/helpers/form.rst]: So should I close the PR or duplicate example?

Comment (markstory) [inline en/views/helpers/form.rst]: Duplicate the example.

**Comment by inoas** [comment]
Good now?

**Comment by markstory** [comment]
Thanks!

**Pull request merged**